<!--igvc_run.launch-->
<launch>

<!-- =============   TRANSFORMS  ======================= -->
<!-- TODO add transforms as a seperate launch file so when they change here they change in all launch files, like bag and demos -->
<include file="$(find igvc_run)/launch/tf.launch"></include>


<include file="$(find igvc_run)/launch/move_base.launch"></include>

<include file="$(find usma_roboteq)/launch/roboteq_driver.launch"></include> 

<!-- =============   VELODYNE  ======================= -->
<include file="$(find velodyne_bringup)/launch/igvc_velodyne.launch"></include>

<!-- =============   LASER SCAN MATCHER  ======================= -->
<node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen"> 
	<remap from="/imu_data" to="/imu/data"/>   
	<param name="cloud" value="/velodyne_points"/>
	<param name="use_cloud_input" value="yes"/>
	<param name="max_iterations" value="10"/>    
	<param name="use_imu" type="bool" value="false" />    
	<param name="use_odom" type="bool" value="false" />    
	<param name="fixed_frame" type="string" value="/map" />    
	<param name="base_frame" type="string" value="/base_footprint" />
</node>

<!-- Start an rviz node with a custom configuration  -->
<node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find igvc_run)/launch/igvc_run_everything.rviz"/>
</launch>

